src/undate/converters/base.py

```diff
@@ -136,9 +137,13 @@ class BaseCalendarConverter(BaseDateConverter):
     #: Converter name. Subclasses must define a unique name.
     name: str = "Base Calendar Converter"
 
-    def max_month(self, year: int) -> int:
-        """Maximum month for this calendar for this year"""
-        raise NotImplementedError
+    def min_month(self) -> int:
+        """First month for this calendar. Defaults to 1."""
+        return 1
+
+    def max_month(self) -> int:
+        """Last month for this calendar. Defaults to 12."""
+        return 12
```

Comment:
Why is min_month / max_month set to 1 and 12 in this base class? Should this not be abstract here and set in the gregorian class?